docs/csharp/tutorials/inheritance.md

```diff
@@ -0,0 +1,294 @@
+
+- Whether it makes sense to instantiate the base class. If it does not, we should apply the [abstract](../language-reference/keywords/abstract.md) keyword to the class. If an attempt is made to instantiate a class marked with the `abstract` keyword by a direct call to its class constructor, the C# compiler generates error CS0144, "Cannot create an instance of the abstract class or interface." If an attempt is made to instantiate the class by using reflection, the reflection method throws a @System.MemberAccessException. Otherwise, our `Publication` class can be instantiated by calling its class constructor.
+
+   By default, a base class can be instantiated by calling its class constructor. Note that we do not have to explicitly define a class constructor. If one is not present in the base class' source code, the C# compiler automatically provides a default (parameterless) constructor.
+
+   For our example, we'll mark the `Publication` class as [abstract](../language-reference/keywords/abstract.md) so that it cannot be instantiated.
+
+- Whether derived classes must inherit the base class implementation of a particular members, or whether they have the option to override the base class implemnetation. We have to use the [virtual](../language-reference/keywords/virtual.md) keyword to allow derived classes to override a base class method. By default, methods defined in the base class are *not* overridable.
+
+- Whether a derived class represents the final class in the inheritance hierarchy and cannot itself be used as a base class for additional derived classes. By default, any class can serve as a base class. We can apply the [sealed](../language-reference/keywords/sealed.md) keyword to indicate that a class cannot serve as a base class for any additional classes. Attempting to derive from a sealed class generated compiler error CS0509, "cannot derive from sealed type <typeName>".
+
+   For our example, we'll mark our derived class as `sealed`.
+
+The following example shows the source code for the `Publication` class, as well as a `PublicationType` enumeration that is returned by the `Publication.PublicationType` property. In addition to the members that it inherits from @System.Object, the `Publication` class defines the following unique members and member overrides:
+
+[!CODE [Inheritance](../../../samples/snippets/csharp/tutorials/inheritance/base-and-derived.cs#1)]
+
+- A constructor
+
+  Because the `Publication` class is `abstract`, it cannot be instantiated directly from code like the following:
+
+  ```cs
+  var publication = new Publication("Tiddlywinks for Experts", "Fun and Games",
+                                    PublicationType.Book);
+  ```
+
+  However, its class constructor can be called directly from derived class constructors, as the source code for the `Book` class shows.
```

Comment:
I would use the term "instance constructor" instead of "class constructor". "Class Constructor" is often used to refer to the static constructor of a type.